fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingScreen.kt

```diff
@@ -268,10 +266,10 @@ private fun defaultPreviewPages() = listOf(
         type = OnboardingPageUiData.Type.DEFAULT_BROWSER,
         imageRes = R.drawable.ic_onboarding_welcome,
         title = stringResource(R.string.juno_onboarding_default_browser_title_nimbus_2),
-        description = stringResource(R.string.juno_onboarding_default_browser_description_nimbus_2),
-        linkText = stringResource(R.string.juno_onboarding_default_browser_description_link_text),
+        description = stringResource(R.string.juno_onboarding_default_browser_description_nimbus_3),
         primaryButtonLabel = stringResource(R.string.juno_onboarding_default_browser_positive_button),
         secondaryButtonLabel = stringResource(R.string.juno_onboarding_default_browser_negative_button),
+        privacyCaption = null,
```

Comment:
❓ lets show the caption in the preview?

Reply:
Sounds good.  I'll add to all of the cards so it's taken into consideration when we change things.  Thanks